src/xg.cpp

```diff
@@ -258,6 +272,163 @@ void XG::load(istream& in) {
     }
 
 }
+    
+void XG::convert_old_edge_to_new(int_vector<> g_iv_old, bit_vector g_bv_old, rank_support_v<1> g_bv_rank_old, bit_vector::select_1_type g_bv_select_old) {
+    size_t g_iv_old_size =
+        node_count * G_NODE_HEADER_LENGTH // record headers
+        + edge_count * 2 * G_EDGE_LENGTH; // edges (stored twice)
+
+    int64_t header = G_NODE_HEADER_LENGTH;
+    int64_t headerfrom = G_NODE_HEADER_LENGTH;
+    int doubly_reversing_sides = 0;
```

Comment:
I don't think you're using the "doubly reversing" terminology the same way it's used in the rest of the code base. I believe you're looking for reversing self-edges here, right? Elsewhere, doubly reversing refers to edges that have both `from_start == true` and `to_end == true`.